src/avram/validations.cr

```diff
@@ -143,11 +144,11 @@ module Avram::Validations
       size = attribute.value.to_s.size
 
       if !min.nil? && size < min
-        attribute.add_error "is too short"
+        attribute.add_error message
```

Comment:
Can we try to keep the previous messages for both of these validations?

```suggestion
        attribute.add_error message || "is too short"
```

The message param would default to nil